pkg/occlient/occlient.go

```diff
@@ -1287,6 +1287,9 @@ func (c *Client) PatchCurrentDC(name string, dc appsv1.DeploymentConfig, prePatc
 		}
 	}
 
+	// copy the any remaining volumes and volume mounts
+	copyVolumesAndVolumeMounts(dc, currentDC, existingCmpContainer)
```

Comment:
Why is this called again? I is already inside the `prePatchDCHandler` that is called a few lines before this.

Reply:
`prePatchDCHandler()` above is only called during local to git updates

Reply:
Wouldn't it be better to remove it from prePatchHandler and call just call it here?